pinkman-rx3/src/main/java/com/redmadrobot/pinkman_rx3/RxPinkman.kt

```diff
@@ -0,0 +1,39 @@
+package com.redmadrobot.pinkman_rx3
+
+import com.redmadrobot.pinkman.Pinkman
+import com.redmadrobot.pinkman.exception.BlacklistedPinException
+import io.reactivex.rxjava3.core.Completable
+import io.reactivex.rxjava3.core.Scheduler
+import io.reactivex.rxjava3.core.Single
+import io.reactivex.rxjava3.schedulers.Schedulers
+
+
+@Throws(BlacklistedPinException::class)
+fun Pinkman.createPinAsync(
+    newPin: String,
+    force: Boolean = false,
```

Comment:
The name of this parameter doesn't make sense for me.
Maybe will be better to call it `overwriteExisting`?